app/assets/javascripts/dashboard.js

```diff
@@ -1,8 +1,19 @@
 
-$(document).on('ContentLoad', function(){start_gridster()});
+$(document).on('ContentLoad', function(){start_gridster(); auto_refresh()});
 
 $(document).on("click",".gridster>ul>li>.close" ,function(){ hide_widget(this);});
 
+function auto_refresh(){
+  var element = $(".auto-refresh");
+  if (element[0]) {
+    setTimeout(function(){
+      if ($(".auto-refresh").hasClass("on")) {
+        window.location.reload(true);
```

Comment:
I don't think so. it will cause a lot of problems IMHO. what's wrong with good ol' refresh?